docker-compose.yml

```diff
@@ -69,3 +69,53 @@ services:
       - "./rabbitmq-data:/var/lib/rabbitmq/mnesia/"
     restart: unless-stopped
 
+  postgres:
+    image: postgres:12-alpine
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_PASSWORD=password
+    volumes:
+      - ./postgres/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+      - ./postgres/script:/script:ro
+      - pg-data:/var/lib/postgresql/data
+
+  pgadmin:
+    image: dpage/pgadmin4:8.6
+    ports:
+      - 5050:5050
+    environment:
+      - PGADMIN_LISTEN_PORT=5050
+      - PGADMIN_DEFAULT_EMAIL=dev@akvo.org
+      - PGADMIN_DEFAULT_PASSWORD=password
+    volumes:
+      - ./postgres/pgadmin4/servers.json:/pgadmin4/servers.json
+      - pg-admin-data:/var/lib/pgadmin
+    depends_on:
+      - postgres
+
+  backend:
+    image: python:3.12.3
+    working_dir: /app
+    environment:
+      - DATABASE_URL=postgresql://akvo:password@postgres:5432/dev
+    command: ["./dev.sh"]
+    volumes:
+      - ./backend:/app:delegated
+    depends_on:
+      - postgres
+
+  frontend:
+    image: akvo/akvo-node-18-alpine:20230831.105309.b9593b7
+    working_dir: /app
+    ports:
+      - 3001:3000
+    command: ["./dev.sh"]
+    volumes:
+      - ./frontend:/app:delegated
+    depends_on:
+      - backend
+
+volumes:
```

Comment:
I'm thinking it will be easier if we use docker compose down -v to reset them. Besides, I also created a script to dump the database [here](https://github.com/akvo/rag-doll/blob/57a259ead036d367e22a2944ddcd6d3b1a7ff1e0/postgres/script/dump-db.sh). This script will override [postgres/docker-entrypoint-initdb.d/001-init.sql](https://github.com/akvo/rag-doll/blob/57a259ead036d367e22a2944ddcd6d3b1a7ff1e0/postgres/docker-entrypoint-initdb.d/001-init.sql), so we can have the same exact example data in the dev database when it's needed.

Reply:
In that case, make the others follow the same pattern: `rabbitmq` and `chromadb`.

Reply:
done